src/components/news-form/news-form.js

```diff
@@ -158,6 +159,16 @@ const NewsForm = ({ article, id }) => {
           </TabPanel>
         ))}
       </form>
+
+      <div className={classes.controlsBlock}>
+        <StandardButton
+          id='back'
+          title={config.buttonTitles.GO_BACK_TITLE}
+          variant='outlined'
+          onClickHandler={() => dispatch(push(config.routes.pathToNews))}
```

Comment:
Make sense to move this dispatch to a separated handler function

Reply:
done